README.md

```diff
@@ -38,11 +38,21 @@ Spiegel User Defined Docs
     }
     ```
 
-2. `filter`
+2. `sieve`
     ```js
     {
-      type: 'filter',
-      regex: '<regex>' // Matches against a DB name
+      id: '_design/sieve',
+      views: {
+        sieve: {
+          map: "function(doc) {
+            if (!doc.key) {
+              return;
+            }
+            if (/dbname1|dbname2/.test(doc.key)) {
+               emit(/:(.*)$/.exec(doc.key))[1]);
```

Comment:
I think I introduced an extra parenthesis there, it should probably be: `emit(/:(.*)$/.exec(doc.key)[1]);`